core/src/main/java/com/linecorp/armeria/server/Server.java

```diff
@@ -162,6 +163,51 @@ public String defaultHostname() {
         }
     }
 
+    /**
+     * Returns the primary {@link ServerPort}'s port that this {@link Server} is listening to.
```

Comment:
`Returns the local {@link ServerPort} that this {@link Server} is listening to.`